front/src/modules/ui/editable-field/components/EditableFieldDisplayMode.tsx

```diff
@@ -73,6 +71,16 @@ type OwnProps = {
   isDisplayModeFixHeight?: boolean;
 };
 
+function displayEmptyIfNothingToShow(
```

Comment:
What is the rationale for a separated method ?
The simple logic could be inline and I'm not sure the naming greatly helps understanding here.

Reply:
@emilienchvt you are right! The function was a bit more complex originally and after simplification I didn't realise inline would be better. I'll change that in a future PR. Thank you!